### ISSUE TITLE
Ticket screen

### ISSUE THREAD

**Issue by tbergq** [kiwicom/mobile#702]
[Zeplin](https://zpl.io/25EY9pn)
![img](https://cdn.zeplin.io/59df4a744f774251779020d9/screens/DDBE1DC2-0681-45A1-8E9C-200E1FA74BEF.png)

## tasks
- [ ] e-ticket
- [ ] boarding pass

**Comment by tbergq** [comment]
@MRTNKCR could you please find out when we offer apple wallet ticket?

**Comment by pistelak** [comment]
@tbergq For now, we are able to offer wallet tickets only for flights with Ryanair or Wizzair.

**Comment by pistelak** [comment]
No, these tickets are part of the response. I believe, that there is a field `pkpass` inside of each passenger and ticket is identified by `routeID`. I'll try to find an example response with them.

**Comment by tbergq** [comment]
Thanx 🙂

**Comment by tbergq** [comment]
@pistelak any progress on this?

**Comment by pistelak** [comment]
@tbergq DM (Slack)
